From 3cd2eab961bf9d34b3dc7dfa89abd3436cb09edb Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Wed, 10 Jan 2018 03:22:52 +0200
Subject: [PATCH 062/249] ether: Add custom TWRP link

* android-8.1 branch built in cm-14.1 with some fixes for cryptfs_hw
* We need a custom recovery for fully working decryption

Change-Id: Iea62e30ef1f7269fb317cefb9ba474ced33cb98d
---
 _data/devices/ether.yml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/_data/devices/ether.yml b/_data/devices/ether.yml
index 10e7f96..22dcf2a 100644
--- a/_data/devices/ether.yml
+++ b/_data/devices/ether.yml
@@ -11,6 +11,7 @@ cpu: Cortex-A53 & Cortex-A57
 cpu_cores: '6'
 cpu_freq: 4 x 1.4 GHz & 2 x 1.8 GHz
 current_branch: 14.1
+custom_twrp_link: https://androidfilehost.com/?fid=817906626617957398
 depth: 7 mm (0.28 in)
 download_boot: With the device powered off, hold <kbd>Volume Up</kbd> + <kbd>Power</kbd>.
 gpu: Adreno 418
-- 
2.17.1

